src/jobs/codeql.yml

```diff
@@ -29,41 +29,49 @@ steps:
       name: Setting up CodeQL
       working_directory: ~/
       command: |
+        mkdir -p ~/bin
```

Comment:
So now this assumes that the directory does not exist, IIUC.
How does the script behave if this commands returns that it was already present?

Reply:
mkdir -p creates all the directories including the leaves. if a directory already exists, it simply returns 0 as the return code, meaning everything is fine. you can't identify if the directory already existed before by the return code and we don't have to.

does this answer your question?

Reply:
I knew about -p but I was not sure it would return 0 in case of a conflict. All good 🙂